### ISSUE TITLE
Add qtc-sourcetrail to Qt plugin repo

### ISSUE THREAD

**Issue by vincent-hui** [CoatiSoftware/qtc-sourcetrail#13]
qtc-sourcetrail can get mechanical adapted to API changes of qtc if you submit qtc-sourcetrail to the Qt Creator source on codereview.qt-project.org and it is accepted there.

**Comment by vincent-hui** [comment]
Thank you very much for open-sourcing Sourcetrail.
Do you have a plan to submit qtc-sourcetrail to the Qt Creator source on codereview.qt-project.org?
That can save maintenance time and help promoting Sourcetrail.

**Comment by mlangkabel** [comment]
Yes, now that we are open-source I think that it makes much more sense to submit the plugin to Qt Creator, since everybody can use it now.
